src/app/beer_garden/api/http/handlers/v1/job.py

```diff
@@ -365,10 +366,18 @@ async def post(self, job_id):
         """
         _ = self.get_or_raise(Job, JOB_CREATE, id=job_id)
 
+        reset_interval = (
+            True if self.get_argument("reset_interval", "False") == "True" else False
+        )
+
         try:
-            await self.client(Operation(operation_type="JOB_EXECUTE", args=[job_id]))
+            await self.client(
+                Operation(operation_type="JOB_EXECUTE", args=[job_id, reset_interval])
+            )
         except ValidationError:
             raise NotFoundError
+        except ModelValidationError:
```

Comment:
You should capture the message here and use it as the `reason` on the `BadRequest`.  Like:

```python
except ModelValidationError as exc:
    raise BadRequest(reason=f"{exc}")
```

The way the `write_error` method works in the `BaseHandler` makes this not useful yet, but once we get around to fixing that handling in the `BaseHandler`, having the reason listed here will make it so that a useful message is returned with the response, as opposed to just "Bad request".